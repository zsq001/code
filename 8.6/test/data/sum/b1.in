100 100
2347106 281205 2196805 1023782 1034152 1018412 701864 1695076 2950188 1620906 2050575 808142 2541620 2071837 719044 1433656 3006408 2185298 355966 1779486 585429 2546171 2541386 2120535 529219 1012886 939786 443892 2512139 841147 246358 401769 845601 1558894 1304018 1099983 308931 1429551 1347746 2568904 806284 2163176 2722253 3063409 655558 2409474 945410 1953849 2060880 3230901 806709 1162559 2089245 2357228 1882388 2328824 1952179 748486 2453279 2486806 2283281 757015 2099855 1887406 205314 213991 3203340 569719 1622828 1609914 1391919 325813 2798081 257718 780233 2099079 194357 887425 1999046 2321206 2501246 3229368 1975739 968419 1754679 3214881 2386932 1773577 2540734 2292546 338066 1710540 726038 2673195 985454 1733491 283661 1482372 2522221 299559 
1 25 72 2856973
2 1 100
1 47 76 1757234
2 1 100
1 15 26 3159536
2 1 100
1 18 60 800850
2 1 100
1 16 44 822587
2 1 100
1 39 45 1581423
2 1 100
1 16 35 1755803
2 1 100
1 22 96 2825292
2 1 100
1 7 20 690594
2 1 100
1 59 71 1552386
2 1 100
1 43 52 554926
2 1 100
1 17 35 1828283
2 1 100
1 26 98 1034552
2 1 100
1 36 73 2991509
2 1 100
1 8 60 2391330
2 1 100
1 50 93 2679484
2 1 100
1 48 77 1602152
2 1 100
1 41 43 2136448
2 1 100
1 64 81 85251
2 1 100
1 29 47 399633
2 1 100
1 40 88 665613
2 1 100
1 29 50 304008
2 1 100
1 19 43 3120287
2 1 100
1 2 64 218653
2 1 100
1 39 57 1906252
2 1 100
1 43 71 977388
2 1 100
1 20 39 2061743
2 1 100
1 14 80 1621438
2 1 100
1 18 43 1640060
2 1 100
1 15 77 570194
2 1 100
1 16 68 1655807
2 1 100
1 2 58 2022261
2 1 100
1 41 65 2485360
2 1 100
1 92 96 1700885
2 1 100
1 21 84 1632002
2 1 100
1 48 60 504424
2 1 100
1 12 91 587576
2 1 100
1 17 93 745722
2 1 100
1 4 43 344423
2 1 100
1 26 30 1330226
2 1 100
1 22 60 2400449
2 1 100
1 70 90 1146273
2 1 100
1 31 41 1788192
2 1 100
1 52 62 1493525
2 1 100
1 9 11 2113181
2 1 100
1 1 51 797932
2 1 100
1 42 76 2856797
2 1 100
1 14 69 816375
2 1 100
1 11 62 1905152
2 1 100
1 62 85 2310395
2 1 100
