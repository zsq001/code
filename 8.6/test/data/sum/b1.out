266116848
347212741
149833696
633773882
734213580
263315699
70623563
223872676
578048527
325188117
44297400
909176606
754656862
211744268
846808815
839888820
210224668
714726022
210444655
250761261
646911587
901154595
614382675
472743940
994792112
729092601
635433610
389686388
921963006
580310156
9210733
523554464
131870943
484922796
691899817
881254143
474648996
428402116
383157465
239661646
885606946
101448850
787882539
927233639
604534430
242859738
487212519
161656364
255475190
276159607
